# OHSU_TMT_Publications

List of publications where TMT labeling was used in the OHSU PSR core. Many papers use the freely available [PAW pipeline](https://github.com/pwilmart/PAW_pipeline). Several of the studies used multiple TMT-plexes and the [IRS method](https://github.com/pwilmart/IRS_validation) to extend isobaric labeling to larger numbers of samples.

The recommended reference to cite for internal reference scaling is Plubell, _et al._, MCP 2017 (**Reference 3 below**).

Some work from other laboratories and institutions using methods/techniques developed in the OHSU proteomic core is also listed.

---

## OHSU Work

1. [Huan, J., Hornick, N.I., Goloviznina, N.A., Kamimae-Lanning, A.N., David, L.L., Wilmarth, P.A., Mori, T., Chevillet, J.R., Narla, A., Roberts, C.T. and Loriaux, M.M., 2015. Coordinate regulation of residual bone marrow function by paracrine trafficking of AML exosomes. Leukemia, 29(12), pp.2285-2295.](https://www.nature.com/articles/leu2015163)

1. [Midgett, M., López, C.S., David, L., Maloyan, A. and Rugonyi, S., 2017. Increased hemodynamic load in early embryonic stages alters endocardial to mesenchymal transition. Frontiers in physiology, 8, p.56.](https://www.frontiersin.org/articles/10.3389/fphys.2017.00056/full)

1. [Plubell, D.L., Wilmarth, P.A., Zhao, Y., Fenton, A.M., Minnier, J., Reddy, A.P., Klimek, J., Yang, X., David, L.L. and Pamir, N., 2017. Extended multiplexing of tandem mass tags (TMT) labeling reveals age and high fat diet specific proteome changes in mouse epididymal adipose tissue. Molecular & Cellular Proteomics, 16(5), pp.873-890.](https://www.mcponline.org/content/16/5/873.short)

1. [Whitson, J.A., Wilmarth, P.A., Klimek, J., Monnier, V.M., David, L. and Fan, X., 2017. Proteomic analysis of the glutathione-deficient LEGSKO mouse lens reveals activation of EMT signaling, loss of lens specific markers, and changes in stress response proteins. Free Radical Biology and Medicine, 113, pp.84-96.](https://www.sciencedirect.com/science/article/abs/pii/S0891584917307724)

1. [Krey, J.F., Dumont, R.A., Wilmarth, P.A., David, L.L., Johnson, K.R. and Barr-Gillespie, P.G., 2018. ELMOD1 stimulates ARF6-GTP hydrolysis to stabilize apical structures in developing vestibular hair cells. Journal of Neuroscience, 38(4), pp.843-857.](https://www.jneurosci.org/content/38/4/843.abstract)

1. [Vranka, J.A., Staverosky, J.A., Reddy, A.P., Wilmarth, P.A., David, L.L., Acott, T.S., Russell, P. and Raghunathan, V.K., 2018. Biomechanical rigidity and quantitative proteomics analysis of segmental regions of the trabecular meshwork at physiologic and elevated pressures. Investigative ophthalmology & visual science, 59(1), pp.246-259.](https://iovs.arvojournals.org/article.aspx?articleid=2670145)

1. [Hulett, T.W., Jensen, S.M., Wilmarth, P.A., Reddy, A.P., Ballesteros-Merino, C., Afentoulis, M.E., Dubay, C., David, L.L. and Fox, B.A., 2018. Coordinated responses to individual tumor antigens by IgG antibody and CD8+ T cells following cancer vaccination. Journal for immunotherapy of cancer, 6(1), p.27.](https://link.springer.com/article/10.1186/s40425-018-0331-0)

1. [Plubell, D.L., Fenton, A.M., Wilmarth, P.A., Bergstrom, P., Zhao, Y., Minnier, J., Heinecke, J.W., Yang, X. and Pamir, N., 2018. GM-CSF driven myeloid cells in adipose tissue link weight gain and insulin resistance via formation of 2-aminoadipate. Scientific reports, 8(1), pp.1-12.](https://www.nature.com/articles/s41598-018-29250-8)

1. [Hegarty, D.M., David, L.L. and Aicher, S.A., 2018. Lacrimal gland denervation alters tear protein composition and impairs ipsilateral eye closures and corneal nociception. Investigative ophthalmology & visual science, 59(12), pp.5217-5224.](https://iovs.arvojournals.org/article.aspx?articleid=2711471)

1. [Wakasaki, R., Matsushita, K., Golgotiu, K., Anderson, S., Eiwaz, M.B., Orton, D.J., Han, S.J., Lee, H.T., Smith, R.D., Rodland, K.D., Piehowski, P.D. and Hutchens, M.P., 2019. Glomerular filtrate proteins in acute cardiorenal syndrome. JCI insight, 4(4).](https://insight.jci.org/articles/view/122130)

1. [Wakeham, C.M., Wilmarth, P.A., Cunliffe, J.M., Klimek, J.E., Ren, G., David, L.L. and Morgans, C.W., 2019. Identification of PKCα-dependent phosphoproteins in mouse retina. Journal of proteomics, 206, p.103423.](https://www.sciencedirect.com/science/article/abs/pii/S1874391919301952)

1. [Rocheleau, A.D., Melrose, A.R., Cunliffe, J.M., Klimek, J., Babur, Ö., Tassi Yunga, S., Ngo, A.T., Pang, J., David, L.L., McCarty, O.J. and Aslan, J.E., 2019. Identification, Quantification, and System Analysis of Protein N‐ε Lysine Methylation in Anucleate Blood Platelets. Proteomics, 19(11), p.1900001.](https://onlinelibrary.wiley.com/doi/abs/10.1002/pmic.201900001)

1. [Abdelhamed, S., Butler, J.T., Doron, B., Halse, A., Nemecek, E., Wilmarth, P.A., Marks, D.L., Chang, B.H., Horton, T. and Kurre, P., 2019. Extracellular vesicles impose quiescence on residual hematopoietic stem cells in the leukemic niche. EMBO reports, 20(7), p.e47546.](https://www.embopress.org/doi/full/10.15252/embr.201847546)

1. [Cheung, C.Y., Anderson, D.F. and Brace, R.A., 2019. Multiomics analyses of vesicular transport pathway-specific transcripts and proteins in ovine amnion: responses to altered intramembranous transport. Physiological genomics, 51(7), pp.267-278.](https://journals.physiology.org/doi/full/10.1152/physiolgenomics.00003.2019)

1. [Svensson, K., LaBarge, S.A., Sathe, A., Martins, V.F., Tahvilian, S., Cunliffe, J.M., Sasik, R., Mahata, S.K., Meyer, G.A., Philp, A., David, L.L., Ward, S.R., McCurdy, C.E., Aslan, J.E. and Schenk, S., 2020. p300 and cAMP response element‐binding protein‐binding protein in skeletal muscle homeostasis, contractile function, and survival. Journal of cachexia, sarcopenia and muscle, 11(2), pp.464-477.](https://onlinelibrary.wiley.com/doi/full/10.1002/jcsm.12522)

1. [Cheung, C.Y. and Brace, R.A., 2020. Altered proteomics profile in the amnion of patients with oligohydramnios. Physiological reports, 8(4), p.e14381.](https://physoc.onlinelibrary.wiley.com/doi/full/10.14814/phy2.14381)

1. [Anand, S., Chatterjee, N. and Espinosa-Diez, C., 2020. A miR-494 dependent feedback loop regulates ER stress. BioRxiv.](https://www.biorxiv.org/content/10.1101/2020.05.12.088856v1.abstract)

---

## Other Institutions:

1. [Contreras-Llano, L.E., Meyer, C., Liu, Y., Sarker, M., Lim, S., Longo, M.L. and Tan, C., 2020. Holistic engineering of cell-free systems through proteome-reprogramming synthetic circuits. Nature communications, 11(1), pp.1-10.](https://www.nature.com/articles/s41467-020-16900-7)
  - Uses PAW pipeline and IRS; SPS-MS3 Fusion Lumos; E. coli engineering

1. [Min, C.W., Park, J., Bae, J.W., Agrawal, G.K., Rakwal, R., Kim, Y., Yang, P., Kim, S.T. and Gupta, R., 2020. In-Depth Investigation of Low-Abundance Proteins in Matured and Filling Stages Seeds of Glycine max Employing a Combination of Protamine Sulfate Precipitation and TMT-Based Quantitative Proteomic Analysis. Cells, 9(6), p.1517.](https://www.mdpi.com/2073-4409/9/6/1517)
  - MaxQuant and IRS; MS2 Q Exactive; Soybean seed Proteomics
1. [Amin, B. and Robinson, R.A., 2020. Dataset of quantitative proteomic analysis to understand aging processes in rabbit liver. Data in Brief, p.105701.](https://www.sciencedirect.com/science/article/pii/S2352340920305953)
  - Proteome Discoverer and IRS; SPS-MS3 Fusion Lumos; Aging changes in rabbit liver
1. [Amin, B., Ford, K.I. and Robinson, R.A., 2020. Quantitative proteomics to study aging in rabbit liver. Mechanisms of Ageing and Development, p.111227.](https://www.sciencedirect.com/science/article/abs/pii/S0047637420300221)
  - Proteome Discoverer and IRS; SPS-MS3 Fusion Lumos; Aging changes in rabbit liver
1. [Bojkova, D., Klann, K., Koch, B. et al. Proteomics of SARS-CoV-2-infected host cells reveals therapy targets. Nature (2020). https://doi.org/10.1038/s41586-020-2332-7](https://www.nature.com/articles/s41586-020-2332-7)
  - Proteome Discoverer/SEQUEST and IRS; MS2 Q Exactive HF; SARS-CoV-2 infected human cells
1. [Espino, J.A., King, C.D., Jones, L.M. and Robinson, R.A., 2020. In Vivo Fast Photochemical Oxidation of Proteins Using Enhanced Multiplexing Proteomics. Analytical Chemistry.](https://pubs.acs.org/doi/full/10.1021/acs.analchem.0c00174)
  - Proteome Discoverer; SPS-MS3 Fusion Lumos; Photochemical oxidative footprinting in _C. elegans_.
1. [Gupta, R., Min, C.W., Kim, S.W., Yoo, J.S., Moon, A.R., Shin, A.Y., Kwon, S.Y. and Kim, S.T., 2020. A TMT-Based Quantitative Proteome Analysis to Elucidate the TSWV Induced Signaling Cascade in Susceptible and Resistant Cultivars of Solanum lycopersicum. Plants, 9(3), p.290.](https://www.mdpi.com/2223-7747/9/3/290)
  - MaxQuant and IRS; MS2 Q Exactive; tomato leaf after TSWV treatment
1. [Afshin, B., Kaushik, C., Homer, F., Hossein, F., Valery, B., Lai, P.S.H., Saravia-Butler, A.M., Hardiman, G., Taylor, D., Galazka, J.M. and Costes, S.V., 2019. Multi-omics analysis of multiple missions to space reveal a theme of lipid dysregulation in mouse liver. Scientific Reports (Nature Publisher Group), 9(1).](https://www.nature.com/articles/s41598-019-55869-2)
  - MaxQuant and IRS; ; mouse livers
1. [Van de Walle, P., Geens, E., Baggerman, G., José Naranjo-Galindo, F., Askjaer, P., Schoofs, L. and Temmerman, L., 2019. CEH-60/PBX regulates vitellogenesis and cuticle permeability through intestinal interaction with UNC-62/MEIS in Caenorhabditis elegans. PLoS biology, 17(11), p.e3000499.]()
  - MaxQuant and IRS; CID/HCD Orbitrap Elite; _C. elegans_ egg development
1. [Wu, M., McCain, J.S.P., Rowland, E., Middag, R., Sandgren, M., Allen, A.E. and Bertrand, E.M., 2019. Manganese and iron deficiency in Southern Ocean Phaeocystis antarctica populations revealed through taxon-specific protein indicators. Nature communications, 10(1), pp.1-10.](https://www.nature.com/articles/s41467-019-11426-z)
  - Proteome Discoverer/SEQUEST and IRS; MS2 Orbitrap Velos Pro; Prymnesiophyceae (algae) mineral deficiencies

---

Phillip Wilmarth<br />OHUS PSR Core<br />July 7, 2020
